Podfile

```diff
@@ -35,7 +35,7 @@ def aztec
     #pod 'WordPress-Editor-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :commit => ''
     #pod 'WordPress-Editor-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :tag => ''
     #pod 'WordPress-Editor-iOS', :path => '../AztecEditor-iOS'
-    pod 'WordPress-Editor-iOS', '~> 1.19.8'
+    pod 'WordPress-Editor-iOS', '1.19.7'
```

Comment:
Ah good catch, I didn't realize that the `Podfile.lock` was actually pointing to the newer version of Aztec 🤦 .